scripts/run.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+password_url="taskcluster/secrets/v1/secret/repo:github.com/mozilla/balrog:dockerhub"
+repo_token=$(curl ${password_url} | python -c 'import json, sys; a = json.load(sys.stdin); print a["secret"]["repo_token"]')
```

Comment:
I think you're going to need to do something to make sure this (and the actual "coveralls" command) only gets run for Taskcluster. You probably should put it inside an "if" block in run_backend_tests, and make it conditional on something you set in https://github.com/mozilla/balrog/blob/master/.taskcluster.yml#L9.